specutils/analysis/location.py

```diff
@@ -14,7 +14,7 @@
 __all__ = ['centroid']
 
 
-def centroid(spectrum, region):
+def centroid(spectrum, regions=None, region=None):
```

Comment:
This would be more backwards compatible.
```suggestion
def centroid(spectrum, region=None, regions=None):
```

Reply:
Good call